inference-engine/tests/ngraph_functions/src/utils/ngraph_helpers.cpp

```diff
@@ -119,10 +119,10 @@ std::vector<std::vector<std::uint8_t>> interpreterFunction(const std::shared_ptr
     for (const auto &result : results) {
```

Comment:
Looks like, we can replace this cycle 

Suggested change: ` for (size_t resultIndex = 0; resultIndex < results.size(); ++i) {`

Reply:
Done.